### PR TITLE
DEVDOCS-2670: Update Config Meta Usage

### PR DESCRIPTION
**Pull request by aglensmith**
# [DEVDOCS-2670](https://jira.bigcommerce.com/browse/DEVDOCS-2670)

## What changed?
* Added lines about config_meta.app.id being optional for create channel requests, but required for select partners.

### CODE REVIEW

File: docs/api-docs/channels/building-channel-apps.md [hunk 1]
```diff
@@ -114,6 +114,8 @@ Once created, channels show up in the product list within the control panel so t
 > **Note**
 >
 > - You can find the app ID in the URL when editing the app in [DevTools](https://devtools.bigcommerce.com/). For more information, see [Finding an App's ID](https://developer.bigcommerce.com/api-docs/apps/tutorials/id).
+> - `config_meta.app.id` is optional; however, if you're building an app that creates or manages a channel, we recommend including the app ID to ensure the user interface in the BigCommerce control panel works properly.
+> - Select partners who are promoted in the Channel Manager must build an app, and include the app ID in the create channel request.
```
Comment (ektravel) [inline docs/api-docs/channels/building-channel-apps.md:118]:
```suggestion
> - Select partners who are promoted in the Channel Manager must build an app, and include the app ID in the create channel request.
```
I find this sentence confusing. Are only select partners promoted in the Channel Manager? Why are they promoted?

File: docs/api-docs/channels/channel-app-requirements.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Once approved, channel apps are discoverable on BigCommerce's App Marketplace. A
 All Partners:
 
 - Uses [Channels API](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api)
-- [Creates a channel](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel) upon app installation using the `app_id`
+- [Creates a channel](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel) upon app installation.
 - Reads and updates channel status via [Channels endpoints](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api)
```
Comment (ektravel) [inline docs/api-docs/channels/channel-app-requirements.md:24]:
```suggestion
- Reads and updates channel status using the [Channels API](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api) endpoints

```

File: docs/api-docs/channels/channel-app-requirements.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ Once approved, channel apps are discoverable on BigCommerce's App Marketplace. A
 All Partners:
 
 - Uses [Channels API](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api)
```
Comment (ektravel) [inline docs/api-docs/channels/channel-app-requirements.md:22]:
```suggestion
- Uses [Channels API](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api).

```

File: docs/api-docs/channels/channel-app-requirements.md [hunk 4]
```diff
@@ -29,6 +29,22 @@ All Partners:
 Select Partners:
 
 - Uses [Big Design](https://developer.bigcommerce.com/big-design/)
```
Comment (ektravel) [inline docs/api-docs/channels/channel-app-requirements.md:31]:
```suggestion
- Uses [Big Design](https://developer.bigcommerce.com/big-design/).

```

**Review by ektravel — COMMENTED** [review]
Added my suggestions.

File: docs/api-docs/channels/building-channel-apps.md (hunk 1 above)
Comment (aglensmith) [reply docs/api-docs/channels/building-channel-apps.md:118]: >Are only select partners promoted in the Channel Manager

Yes.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**